Formula/ripgrep-all.rb

```diff
@@ -3,6 +3,7 @@ class RipgrepAll < Formula
   homepage "https://github.com/phiresky/ripgrep-all"
   url "https://github.com/phiresky/ripgrep-all/archive/v0.9.6.tar.gz"
   sha256 "8cd7c5d13bd90ef0582168cd2bef73ca13ca6e0b1ecf24b9a5cd7cb886259023"
+  license "AGPL-3.0"
```

Comment:
`AGPL-3.0-or-later`

Reply:
Yeah, we can start a thread discussing about (will reply tomorrow with my ideas on this), I did intentionally let it be in this way.

site note, we have some formula with syntax as `AGPL-3.0+`

Reply:
Yeah. We've got several different conventions going at the same time so I think we just need to decide on which one we want to use. We can discuss it tomorrow.